extension/parquet/include/parquet_reader.hpp

```diff
@@ -178,10 +179,16 @@ class ParquetReader : public BaseFileReader {
 	idx_t NumRowGroups() const;
 
 	const duckdb_parquet::FileMetaData *GetFileMetadata() const;
+	string static GetUniqueFileIdentifier(const duckdb_parquet::EncryptionAlgorithm &encryption_algorithm);
 
 	uint32_t Read(duckdb_apache::thrift::TBase &object, TProtocol &iprot);
+	uint32_t ReadEncrypted(duckdb_apache::thrift::TBase &object, TProtocol &iprot, uint16_t row_group_ordinal = 0,
+	                       uint16_t col_idx = 0, uint8_t module = 0, uint16_t page_ordinal = -1);
```

Comment:
Instead of adding 4 parameters, could we pass a struct instead?
```C++
struct MyStruct {
    uint16_t row_group_ordinal = 0;
    uint16_t col_idx = 0;
    uint8_t module = 0;
    uint16_t page_ordinal = -1;
};
```
That keeps the API a bit more clean